assets/styles/style.scss

```
@@ -269,12 +277,23 @@ $color-secondary: #c4c4c4;
     }
   }
 
-  .flickity-page-dots {
+  .flickity-prev-next-button {
+    top: auto;
+    width: 28px;
+    height: 28px;
+    @media screen and (min-width: $screen-sm-min) {
+      top: 50%;
+    }
+  }
+
+  .flickity-page-dots,
+  .flickity-prev-next-button {
     margin: 0;
     padding: 0;
     outline: none;
   }
 
+  .flickity-prev-next-button,
   .flickity-page-dots .dot {
```

For consistency, perhaps we should keep the same selectors order as in lines 289-290.
```suggestion
    .flickity-page-dots .dot,
    .flickity-prev-next-button {
```